tests/testthat/test-convGDX2mif.R

```diff
@@ -28,36 +28,56 @@ test_that("Test if REMIND reporting is produced as it should and check data inte
   }
 
   # finds for each AMT scenario the most recent, successfully converged GDX
-  .findAMTgdx <- function(gdxPaths = NULL) {
+  .findAMTgdx <- function(gdxPaths = NULL, scenario = NULL) {
     didremindfinish <- function(fulldatapath) {
       logpath <- paste0(stringr::str_sub(fulldatapath, 1, -14), "/full.log")
       return(file.exists(logpath) &&
         any(grep("*** Status: Normal completion", readLines(logpath, warn = FALSE), fixed = TRUE)))
     }
-    gdx <- Sys.glob("/p/projects/remind/modeltests/output/*/fulldata.gdx")
+    gdx <- Sys.glob(paste0("/p/projects/remind/modeltests/remind/output/", scenario, "*/fulldata.gdx"))
```

Comment:
this expects you are on the cluster, right?

Reply:
Yes, the AMT test is skipped if you are not on the cluster.